### PR TITLE
BZ2000290:  The PVC will be generated based on the default storage class. The default storage class may provide RWO volumes

### PR DESCRIPTION
**Pull request by agantony**
…ere.adoc

Version(s):
4.8+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2000290

Link to docs preview:
https://54838--docspreview.netlify.app/openshift-enterprise/latest/registry/configuring_registry_storage/configuring-registry-storage-vsphere.html#registry-configuring-storage-vsphere_configuring-registry-storage-vsphere

QE review:
- [x] QE has approved this change.

Additional information:
Merge into `main` and cherry-pick into:
- `enterprise-4.13`
- `enterprise-4.12`
- `enterprise-4.11`
- `enterprise-4.10`
- `enterprise-4.9`
- `enterprise-4.8`

### CODE REVIEW

**Comment by agantony** [comment]
QE review needed: https://bugzilla.redhat.com/show_bug.cgi?id=2000290#c7

File: modules/registry-configuring-storage-vsphere.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ storage:
     claim: <1>
 ----
 +
-<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` PVC.
```
Comment (stevsmit) [inline modules/registry-configuring-storage-vsphere.adoc:90]: swap/ will / is

We try to stay away from future tense/will. 

https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense

**Comment by stevsmit** [comment]
@agantony One nit. Looks good overall.

File: modules/registry-configuring-storage-vsphere.adoc [hunk 2]
```diff
@@ -87,7 +87,7 @@ storage:
...
+<1> Leave the `claim field` blank to allow the automatic creation of an `image-registry-storage` Persistent Volume Claim (PVC). The PVC will be generated based on the default storage class. However, be aware that the default storage class might provide ReadWriteOnce (RWO) volumes, such as a RADOS Block Device (RBD), which can cause issues when replicating to more than one replica.
```
Comment (stevsmit) [inline modules/registry-configuring-storage-vsphere.adoc:90]: swap/ will be / is

We try to stay away from future tense/will. 

https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense

Comment (agantony) [reply modules/registry-configuring-storage-vsphere.adoc:90]: Sure, will do that!

Comment (agantony) [reply modules/registry-configuring-storage-vsphere.adoc]: Done

File: modules/registry-configuring-storage-vsphere.adoc [hunk 3]
```diff
@@ -87,7 +87,7 @@ storage:
     claim: <1>
 ----
 +
-<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` PVC.
+<1> Leave the `claim field` blank to allow the automatic creation of an `image-registry-storage` Persistent Volume Claim (PVC). The PVC is generated based on the default storage class. However, be aware that the default storage class might provide ReadWriteOnce (RWO) volumes, such as a RADOS Block Device (RBD), which can cause issues when replicating to more than one replica.
```
Comment (xiuwang) [inline modules/registry-configuring-storage-vsphere.adoc:90]: Should only shadow `claim`.

Comment (agantony) [reply modules/registry-configuring-storage-vsphere.adoc]: ack ...done

File: modules/registry-configuring-storage-vsphere.adoc [hunk 4]
```diff
@@ -87,7 +87,7 @@ storage:
     claim: <1>
 ----
 +
-<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` PVC.
+<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` Persistent Volume Claim (PVC). The PVC is generated based on the default storage class. However, be aware that the default storage class might provide ReadWriteOnce (RWO) volumes, such as a RADOS Block Device (RBD), which can cause issues when replicating to more than one replica.
```
Comment (snarayan-redhat) [inline modules/registry-configuring-storage-vsphere.adoc:90]:
```suggestion
<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` persistent volume claim (PVC). The PVC is generated based on the default storage class. However, be aware that the default storage class might provide ReadWriteOnce (RWO) volumes, such as a RADOS Block Device (RBD), which can cause issues when replicating to more than one replica.
```

**Review by snarayan-redhat — COMMENTED** [review]
just one small nit

**Pull request merged**